MapboxCoreNavigation/RouteController.swift

```diff
@@ -998,31 +1026,41 @@ extension RouteController {
 
         return event.id.uuidString
     }
-    
-    func enqueueFoundFasterRouteEvent() -> String {
+
+    private func enqueueFoundFasterRouteEvent() -> String {
         let timestamp = Date()
-        let eventDictionary = events.navigationRerouteEvent(routeController: self, eventType: FasterRouteFoundEvent)
-        
+        let eventDictionary = eventsManager.navigationRerouteEvent(routeController: self, eventType: FasterRouteFoundEvent)
+
         sessionState.lastRerouteDate = timestamp
-        
+
         let event = RerouteEvent(timestamp: Date(), eventDictionary: eventDictionary)
-        
+
         outstandingFeedbackEvents.append(event)
-        
+//        eventsManager.enqueueEvent(withName: MMEEventTypeNavigationReroute, attributes: event.eventDictionary)
```

Comment:
@akitchen will we be getting rid of this, or was this commented out for a reason?